onwuvic/MyDiary
# #158909082 Create user login page

#### What does this PR do?
A User login page Interface

#### Description of Task to be completed?
As a User, I should be able to login to the application so that I can have access to create, edit and modify my diaries/entries

#### How should this be manually tested?
Given A User
When User navigates to the login page
Then User should see a simple and descriptive form with email/username and password fields

#### Any background context you want to provide?
This Pull Request should be merged first before #1 to avoid merge conflict

#### What are the relevant pivotal tracker stories?
#158909082
